debian/changelog

```diff
@@ -1,3 +1,16 @@
+authd (0.5.0) questing; urgency=medium
+
+  * Use SQLite instead of bbolt
+  * Use lowercase usernames
+  * Fix races in the PAM module.
+  * Do not support device authentication mode in polkit.
+  * Don't overwrite shell when updating user entry
+  * Improve warning logged when home directory has incorrect owner/group
+  * Support logging to the journal with priority "notice"
+  * More useful error messages
+
```

Comment:
One more thing... Since this is i the _debian_ changelog in theory (and for good practice for when we go to distro), we should include here all changes that touched debian side so basically, stuff in ` git log v0.4.1..HEAD -- debian`

Maybe also mention:
  - The go toolchain version we use now
  - Do not clear terminal context on sudo or other authenticated ops (as per #840)
  - Fixed logging with some locales (as per https://github.com/ubuntu/authd/issues/865)
  - Simplified GDM PAM protocol
  - More integration tests and improved their reliability

Reply:
> One more thing... Since this is i the debian changelog in theory (and for good practice for when we go to distro), we should include here all changes that touched debian side

Does that mean we should *only* mention changes on the debian side + a generic "New upstream version ..."?

Reply:
> Fixed logging with some locales (as per https://github.com/ubuntu/authd/issues/865)

Are we sure that was really an issue caused by authd and that we fixed it?

Reply:
I added `* Fix login via GDM with some non-English locales`